docs/pages/pack/docs/migrating-from-webpack.mdx

```diff
@@ -3,22 +3,85 @@ title: Migrate from webpack to Turbopack
 description: Learn about how to migrate from webpack to its Rust-powered successor, Turbopack.
 ---
 
+import Callout from "../../../components/Callout";
+
 # Migrating from webpack to Turbopack
 
+<Callout type="info">
+  Turbopack now implements basic webpack loader support and configuration familiar to webpack users
+</Callout>
+
 We're planning Turbopack as the successor to webpack. In the future, we plan to give Turbopack all the tools needed to support your webpack app.
 
-Currently, migrating to Turbopack from webpack is **not yet possible**. In the future, we're planning to offer a smooth migration path for all webpack users to join the Turbopack future.
+Currently, Turbopack supports a subset of webpack's loader API and offers similar configuration aliasing module resolution. Note that using webpack-based Next.js plugins as-is from `next.config.js` is **not yet possible**.
+
+## webpack loaders
```

Comment:
Do we support webpack without Next? If not, let's update this heading to stress that this is for Next
```suggestion
## webpack loaders for Next.js
```